src/mlpack/methods/pca/decomposition_policies/robust_svd_method.hpp

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file robust_svd_method.hpp
+ * @author Charan Reddy
+ *
+ * Implementation of the robust svd method for use in the Principal
+ * Components Analysis method.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_PCA_DECOMPOSITION_POLICIES_ROBUST_SVD_METHOD_HPP
+#define MLPACK_METHODS_PCA_DECOMPOSITION_POLICIES_ROBUST_SVD_METHOD_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/robust_svd/robust_svd.hpp>
+
+namespace mlpack {
+namespace pca {
+
+/**
+ * Implementation of the robust SVD policy.
```

Comment:
Can you clarify some more in the documentation here about what robust SVD is, and can you add a reference to the paper you are implementing?